### PR TITLE
Skeleton template for our Micro services

### PR DESCRIPTION
**Pull request by karayanni**
Used the accepted way of defining routes and resources using flask_restful.
This is only the first step but still let me know WDYT and if you have suggestions for improvements

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from flask import Flask
+from flask_restful import Api
+from ApiResources.UserContactInfoApiResource import UserContactInfoApiResource
+from ApiResources.UserContactInfoListApiResource import UserContactInfoListApiResource
+
+app = Flask(__name__)
+api = Api(app)
+
+# here we register the routes by binding them with resources - naming by convention for get/post..
+api.add_resource(UserContactInfoApiResource, "/contactInfo/<string:userId>")
```
Comment (satyamsharma) [inline main.py:10]: I like the aesthetics of decorator
e.g.
@app.route("/contactInfo/<userId>", methods=["GET"])

but we can discuss on Wednesday what everyone thinks

File: test.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# TODO: add some "smart" test
+base = 'http://127.0.0.1:5000/'
+
+response = requests.get(base + 'contactInfo')
+print(response.json())
+
+response = requests.get(base + 'contactInfo/123')
+print(response.json())
+
+response = requests.post(base + 'contactInfo/', {"Email": "nice", "Id": 231, "phoneNumber": 9294612209})
```
Comment (satyamsharma) [inline test.py:13]: we need to specify that it is a application/json type data

**Review by satyamsharma — CHANGES_REQUESTED** [review]
image

post seems to return a 400 on my end, possibly because of a missing header

also, there is a redundant status 308 in the beginning when I ran test.py

File: test.py (hunk 2 above)
Comment (karayanni) [reply test.py:13]: I think it infers that automatically as the test is passing

Comment (karayanni) [reply test.py:13]: I added it explicitly as I agree it is better to explicit and checked that the header is added:
{'Content-Type': 'application/json', 'Content-Length': '41', 'Server': 'Werkzeug/2.0.3 Python/3.10.4', 'Date': 'Wed, 12 Oct 2022 14:20:34 GMT'}

**Pull request merged**